go.mod

```diff
@@ -29,5 +29,6 @@ require (
 replace (
 	github.com/Shopify/sarama => github.com/elastic/sarama v1.19.1-0.20200629123429-0e7b69039eec
 	github.com/dop251/goja => github.com/andrewkroh/goja v0.0.0-20190128172624-dd2ac4456e20
+	github.com/elastic/go-elasticsearch/v8 => github.com/blakerouse/go-elasticsearch/v8 v8.0.0-20210413134915-cee76d43c5cf
```

Comment:
@blakerouse Should we get it in directly with this "fork"? I think it is unlikely that we can use a stable release soon enough.

Reply:
@ruflin really? I just pushed this so others can test it. I didn't think getting it merged into go-elasticsearch was going to be a big deal. Its a very small change.

Reply:
Looks like I was wrong and it is already in!

Reply:
Yep, updated to use new upstream version.